src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -25,8 +25,50 @@ const DEFAULT_RECONNECT_LIMIT = 10;
 
 const NOOP = () => {};
 
+let maxCallstackSize;
+
 // -- Local methods section --
 
+function getMaxCallstackSize() {
+    if (maxCallstackSize) {
+        return maxCallstackSize;
+    }
+
+    let index = 0;
+    function recurse() {
+        index++;
+        recurse();
+    }
+
+    try {
+        recurse();
+    } catch (e) {
+        maxCallstackSize = index;
+    }
+
+    return maxCallstackSize;
+}
+
+function getJSONPayloadString(payloadBuffer) {
+    const chunkSize = getMaxCallstackSize();

```

Comment:
If we decide to use staticly defined max call stack, please validate it against mobile. For example form me, payload of 1000 entires is crashing my chrome on my 6 gb anroid phone. 

![image](https://user-images.githubusercontent.com/117785/84496068-8f7e6880-acac-11ea-9abb-95aa1346c0cf.png)

Test run:
https://jsperf.com/animuj-fromcharcode

Reply:
https://jsperf.com/yugk-fromcharcode
I have tested with 4gb android phone
works fine with 30k entries and chunksize of 500, 1000, 1500